<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPlayListPop</class>
 <widget class="QWidget" name="AddPlayListPop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background:rgb(255,255,255);
	border-radius:10px;
	color:rgb(34,34,34);
	border:none;
	border:2px solid #222222;
}</string>
  </property>
  <widget class="QWidget" name="widget_All" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>210</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>210</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>210</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border-radius:5px;
	font: 10pt &quot;微软雅黑&quot;;	
	color:rgb(255,255,255);
	background:#222222;
}
QLineEdit
{
	padding:5px;
	font: 9pt &quot;微软雅黑&quot;;	
}</string>
   </property>
   <widget class="QLabel" name="label_NewPlayList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;黑体&quot;;
border:none;</string>
    </property>
    <property name="text">
     <string>新建歌单</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>85</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid #222222;
border-radius:5px;</string>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
    <property name="placeholderText">
     <string>请输入歌单名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Confirm">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background:#00BFFF;
	border:none;
}
QPushButton:disabled
{
	background:rgb(193, 193, 193);
}
QPushButton:hover
{
	background-color: rgb(0, 145, 218);
}</string>
    </property>
    <property name="text">
     <string>创建</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Cancle">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover
{
	background:rgb(65, 65, 65);
	border:none;
}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>8</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/icon/logo.png</normaloff>:/icon/logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_Warn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>211</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
font: 8pt &quot;微软雅黑&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>歌单名不可包含空格！</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Warn_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>211</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
font: 8pt &quot;微软雅黑&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>请输入不超过30个字符</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
